clients/src/test/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImplTest.java

```diff
@@ -1256,7 +1249,8 @@ public void testUnresolvedTargetAssignmentIsReconciledWhenMetadataReceived() {
 
         // Assignment should have been reconciled.
         Set<TopicPartition> expectedAssignment = Collections.singleton(new TopicPartition(topicName, 1));
-        verify(subscriptionState).assignFromSubscribed(expectedAssignment);
+        HashSet<TopicPartition> expectedSet = new HashSet<>(expectedAssignment);
```

Comment:
similar to the comment above, can we just verify against the expectedAssignment?

Reply:
Yes, fixed now